packages/lesswrong/components/hooks/useUnreadNotifications.ts

```diff
@@ -69,8 +69,10 @@ export function useUnreadNotifications(): {
   const unreadPrivateMessages = data?.unreadNotificationCounts?.unreadPrivateMessages ?? 0;
   const checkedAt = data?.unreadNotificationCounts?.checkedAt || null;
 
-  const refetchIfNewNotifications = useCallback((timestamp: Date) => {
-    if (!checkedAt || timestamp > checkedAt) {
+  const refetchIfNewNotifications = useCallback((messageBlob: string) => {
+    // TODO: figure out what's going on with messageBlob (which was formerly timestamp, but,
```

Comment:
Pretty sure Jim's PR here has a fix for some incorrect typing going on: https://github.com/ForumMagnum/ForumMagnum/pull/7828